### DISCUSSION TITLE
How to access env variables in cloudflare workers?

### DISCUSSION THREAD

**Discussion by madroneropaulo** [wevm/frog-archived#96] [Q&A] [answered]
I created a frog instance with `npm init frog -- -t cloudflare-worker`
How can I access env variables from wrangler.toml?
I can't access it through hono's env because the route doesn't receive a hono context.
I can't access it through the request.

**Comment by jxom** [comment]
Ah, we can expose [`.env`](https://hono.dev/api/context#env), hold tight

**Reply by jxom** [reply, marked as answer]
Will be in 0.4.3 once build passes.

**Reply by madroneropaulo** [reply]
Thanks!
